### ISSUE TITLE
Stopped working with phpunit 6.3

### ISSUE THREAD

**Issue by poisa** [LimeDeck/phpunit-detailed-printer#6]
The same exact config with 5.7.21 works fine but with 6.3 returns:

```
PHPUnit 6.3.0 by Sebastian Bergmann and contributors.

Could not use "LimeDeck\Testing\Printer" as printer.
```
For what I see they removed the long class names and everything is now namespaced but I don't know if this has anything to do with this problem. Phpunit is not giving any details...

**Comment by poisa** [comment]
I've made a pull request (#7) that makes the class compatible with PHPUnit 6. Since everything is namespaced it will definitely break compatibility with earlier versions so if this pull request is accepted, it might be wise to tag it accordingly.

Let me know if there is anything else I can do.

**Comment by jakubhomoly** [comment]
This has been resolved in 3.0.0.

PHPUnit < 6.0 will still work with ^2.0.0

Thanks

**Comment by robertmain** [comment]
Looks like it's not working with PHPUnit 7

    A PHP Error was encountered

    Severity:    Compile Error
    Message:     Declaration of LimeDeck\Testing\Printer::addError(PHPUnit\Framework\Test $test, Exception $e, $time) must be compatible with PHPUnit\Framework\TestListener::addError(PHPUnit\Framework\Test $test, Throwable $t, float $time): void
    Filename:    C:\Users\robert\Code\PHP\codeigniter-starter\vendor\limedeck\phpunit-detailed-printer\src\Printer.php
    Line Number: 226
    Backtrace:

    Script phpunit --testsuite models handling the test:models event returned with error code 255
